.github/workflows/approve-integration-tests.yml

```diff
@@ -0,0 +1,86 @@
+name: Approve Integration Tests
```

Comment:
Very minor, but maybe we should change `approve-` to `allow-` or `run-`. `Approve` makes it seem like this was bypassing the tests.

```suggestion
name: Approve Integration Tests
```

Reply:
I've updated it to `!run-integration-tests` to also keep consistent with the `run_integration_tests` workflow input.

Reply:
ty!